src/parsing/parsers/generic/jison/insert/insert.test.json

```diff
@@ -9,6 +9,16 @@
       "lowerCase": false
     }
   },
+  {
+    "namePrefix": "should not report errors",
+    "beforeCursor": "INSERT INTO bla.boo (column) VALUES (1, 2, 'a', 3); ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["SELECT"],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
```

Comment:
Let's add a test case when we're still writing a query - `INSERT INTO bla.boo (column) |`.

Reply:
Also i'd like a test case when we insert with multiple columns `INSERT INTO blaa.boo (column1, column2) VALUES (1, 2);`. Just to be sure.